src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/MetricAggregationDeserializer.cs

```diff
@@ -14,7 +14,7 @@ public MetricAggregationDeserializer(ILogger<MetricAggregationDeserializer> logg
         {
         }
 
-        public override MetricAggregationV1 Deserialize(YamlMappingNode node)
+        public override MetricAggregationV1 Deserialize(YamlMappingNode node, IErrorReporter errorReporter)
```

Comment:
Is there a reason why we are not using the new syntax here?

Reply:
I'll update it in another PR.